cineast-api/src/main/java/org/vitrivr/cineast/api/Main.java

```diff
@@ -27,12 +27,26 @@ public static void main(String[] args) {
       }
     }
 
-    /* Start Cineast API endpoint. */
-    APIEndpoint.getInstance().start();
-    GRPCEndpoint.start();
+    /* Start API endpoint. */
+    try {
+      APIEndpoint.getInstance().start();
+    } catch (Throwable e) {
+      System.err.println("Failed to initialize API endpoint due to an exception: " + e.getMessage());
+    }
+
+    /* Start gRPC endpoint. */
+    try {
+      GRPCEndpoint.start();
```

Comment:
If you do error checking here, the block in https://github.com/vitrivr/cineast/blob/cecc4ac9c69f4806a1ab04d9c9888d432fd6eb49/cineast-api/src/main/java/org/vitrivr/cineast/api/GRPCEndpoint.java#L36 can be removed (and should be, as failing silently is never a good strategy).

Reply:
Done!